### ISSUE TITLE
Tasks `build` and `runIde` fail when increasing `org.jetbrains.intellij` to version 1.10.0

### ISSUE THREAD

**Issue by xmasux** [JetBrains/gradle-intellij-plugin#1221]
When upgrading the gradle plugin 'org.jetbrains.intellij' to version 1.10.0 we encounter the following error when executing `./gradlew runIde` and no IDE is started:
* Executed with Java 11/17 for Intel
```
Error: Could not find or load main class 
Caused by: java.lang.ClassNotFoundException:
```
* Executed with Java 11/17 for Mac M1
```
[gradle-intellij-plugin :intellij:runIde] Cannot download JetBrains Java Runtime 'jbr_jcef-11_0_6-osx-aarch64-b520.66'
Unrecognized VM option 'UseConcMarkSweepGC'
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit
```

When downgrading the gradle plugin 'org.jetbrains.intellij' to version 1.9.0 the IDE is started regardless of the Java Runtime.

See the attached minimal example that is based on the [IntelliJ Platform Plugin Template](https://github.com/JetBrains/intellij-platform-plugin-template)
[intellij-platform-plugin-template.zip](https://github.com/JetBrains/gradle-intellij-plugin/files/10086284/intellij-platform-plugin-template.zip)

The problem might be related to the build version that is configured. Everything works fine with version 1.10.0 and the following version setting:
```
pluginSinceBuild = 213
pluginUntilBuild = 223.*
platformType = IC
platformVersion = 2021.3.3
```
But when decreasing the version settings to: 
```
pluginSinceBuild = 193
pluginUntilBuild = 223.*
platformType = IC
platformVersion = 2019.3.5
```
the errors described above occurs.

Performed on:
Macbook Pro Apple M1 Chip
macOS Monterey

The same issue was also filed [here](https://youtrack.jetbrains.com/issue/IJSDK-1592)

**Comment by hsz** [comment]
This happens because you use the `aarch64` runtime for running the Gradle IntelliJ Plugin. Because of that, the architecture for fetched JBR is set to `aarch64` as well: `jbr_jcef-11_0_6-osx-aarch64-b520.66`.

Unfortunately, such an old JBR wasn't released in this architecture, so it's unavailable for downloading.
To handle that on M1 processors, you may specify the JBR architecture now with manually as below:

```
tasks {
  runIde {
    jbrArch.set("x64")
  }
}
```

**Comment by bric3** [comment]
@hsz Might be a nice touch to have a log that hints about the missing architecture instead of `Unrecognized VM option 'UseConcMarkSweepGC'`. E.g.

```diff
  [gradle-intellij-plugin :] Cannot download JetBrains Java Runtime 'jbr_jcef-11_0_7-osx-aarch64-b944.20'
  [gradle-intellij-plugin :] Cannot download JetBrains Java Runtime 'jbr_jcef-11_0_7-osx-aarch64-b944.20'
- Unrecognized VM option 'UseConcMarkSweepGC'
+ Architecture 'aarch64` not available for '11.0.7' (you may override the detected one with 'jbrArch.set("x64")'
  Error: Could not create the Java Virtual Machine.
  Error: A fatal exception has occurred. Program will exit.
```
